packages/code-du-travail-modeles/src/modeles/conventions/1505_commerces_detail_fruits_legumes/__tests__/indemnite-licenciement/formule.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { IndemniteLicenciementPublicodes } from "../../../../../publicodes";
+
+const engine = new IndemniteLicenciementPublicodes(
+  modelsIndemniteLicenciement,
+  "1505"
+);
+
+describe("Formule de l'indemnité de licenciement - CC 1505", () => {
+  describe("Licenciement pour inaptitude non professionnelle", () => {
+    test.each`
+      seniority | expectedFormula                            | expectedExplanations
+      ${7 / 12} | ${""}                                      | ${[]}
+      ${0.67}   | ${"1/4 * Sref * A"}                        | ${["A : Ancienneté totale (0.67 an)", "Sref : Salaire de référence (2800 €)"]}
+      ${18}     | ${"(1/4 * Sref * A1) + (1/3 * Sref * A2)"} | ${["A1 : Ancienneté de 10 ans ou moins (10 ans)", "A2 : Ancienneté au-delà de 10 ans (8 ans)", "Sref : Salaire de référence (2800 €)"]}
+    `(
+      "Avec une ancienneté $seniority ans, un salaire de référence $salaireRef € => $expectedFormula",
```

Comment:
Je crois que tu peux enlever salaire de référence :) 
```suggestion
      "Avec une ancienneté $seniority ans => $expectedFormula",
```